services/wireguard/network/network_darwin.go

```diff
@@ -28,12 +28,17 @@ func (n *network) Provider() (Provider, error) {
 	return Provider{}, errNotSupported
 }
 
-// Consumer adds service consumer public key to the list of allowed peers.
+// AddConsumer adds service consumer public key to the list of allowed peers.
 // It returns information required to configure a consumer instance to allow connections.
-func (n *network) Consumer() (Consumer, error) {
+func (n *network) AddConsumer() (Consumer, error) {
 	return Consumer{}, errNotSupported
 }
 
+// Consumer establish network connection between provider and consumer.
```

Comment:
I really don't get those two functions Consumer() and AddConsumer. Either naming is bad, or they are not doing what they need to do.

Reply:
As I understand Consumer and Provider represent configuration passed to service consumer? Than those structs should be named accordingly or maybe gathered to separate package

Reply:
It was completely refactored.